profiles/base.yaml

```diff
@@ -193,6 +193,7 @@ crossing_training_requires_movement:
 - Trading
 forage_item:
 braid_item:
+collect_item: rock
```

Comment:
Looks like crossing-training was already using forage_item, so I set a default value and updated combat-trainer to use the same setting